Calling/BlobStorageHelperInfo.cs

```diff
@@ -7,14 +11,158 @@ public class BlobStorageHelperInfo
         public string Uri { set; get; }
     }
 
-    public struct RecordingInfo
+    public struct RecInfo
     {
-        public RecordingInfo(string id, string uri)
+        public RecInfo(string id, string uri)
         {
             recordingId = id;
             recordingUri = uri;
         }
         public string recordingId { get; set; }
         public string recordingUri { get; set; }
     }
+
+    static class FileDownloadType
```

Comment:
In general it seems like we should be splitting out these objects into different files? They are just in one file for convenience?

Reply:
This allows us to to have an opportunity to comment what these different parts are for?

Reply:
Separated classes into individual code files